File: beginner/__1000__/1011/1011.php
<?php 
    $r = (float) readline();
    $pi = 3.14159;

    $volume = ($r * $r * $r) * $pi * (4.0/3.0);
    $volume = number_format($volume, 3, ".", '');

    echo "VOLUME = $volume\n";

?>

File: beginner/__1000__/1011/1011.rs
use std::io;

fn volume_esfera(raio: f64) -> f64 {

    const PI: f64 = 3.14159;
    (4.0 / 3.0) * PI * f64::powf(raio, 3.0)
} 

fn main() {

    let mut input = String::new();

    io::stdin().read_line(&mut input).unwrap();
    let raio: f64 = input.trim().parse::<f64>().unwrap_or_default();

    let volume: f64 = volume_esfera(raio);

    println!("VOLUME = {:.3}", volume);
}